informer/integration/test/collector/config_map.go

```diff
@@ -0,0 +1,77 @@
+// +build k8srequired
+
+package collector
```

Comment:
All the helpers are basically copy paste from the other tests and makes refactoring later into common libraries easier.